vagrant/base/devstack/Vagrantfile

```diff
@@ -33,6 +33,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   # Creates a devstack from a base Ubuntu 16.04 image for virtualbox
   config.vm.box = "boxcutter/ubuntu1604"
+  config.vm.box_url = 'https://s3-us-west-1.amazonaws.com/stanford-vagrant/boxcutter-ubuntu1604.box'
```

Comment:
If it's not already, can you split this out into its own commit?
And give it a descriptive commit message, including why we needed to do this in the first place.

Then I'll cherry-pick this into my PR [1].

[1] https://github.com/stvstnfrd/configuration/pull/19

Reply:
It seems like it's already its own commit: "Use different base box".

Reply:
Cherry-picked.